File: proba.py
from sfml import sfml as sf

socket = sf.network.UdpSocket()
recipient = sf.network.IpAddress.get_local_address()
port = 54000
while True:
  if sf.window.Keyboard.is_key_pressed(sf.window.Keyboard.UP):
    socket.send(b"U\0", recipient, port)
  elif sf.window.Keyboard.is_key_pressed(sf.window.Keyboard.DOWN):
    socket.send(b"D\0", recipient, port)
  elif sf.window.Keyboard.is_key_pressed(sf.window.Keyboard.LEFT):
    socket.send(b"L\0", recipient, port)
  elif sf.window.Keyboard.is_key_pressed(sf.window.Keyboard.RIGHT):
    socket.send(b"R\0", recipient, port)

File: proba.cpp
#include <SFML/Graphics.hpp>
#include <SFML/Network.hpp>
#include <SFML/Window.hpp>
#include <dirent.h>
#include <iostream>
#include <vector>

using namespace std;

int main() {
  sf::UdpSocket socket;

  // bind the socket to a port
  // if (socket.bind(54000) != sf::Socket::Done) {
  //     cout<<"Socket binding failed"<<endl;
  // }

  char data[100] = "siema";
  sf::IpAddress recipient = sf::IpAddress::getLocalAddress();
  unsigned short port = 54000;
  if (socket.send(data, 100, recipient, port) != sf::Socket::Done) {
      cout<<"Socket sending failed"<<endl;
  }
  return 0;
}
